.github/workflows/gh-ci.yaml

```diff
@@ -35,6 +36,8 @@ jobs:
 
       - id: get-compatible-python
         uses: MDAnalysis/mdanalysis-compatible-python@main
+        with:
+          release: "latest"
```

Comment:
That's the current py 3.12 hack... https://github.com/MDAnalysis/cookiecutter-mdakit/pull/93